src/components/common/Button/index.tsx

```diff
@@ -8,15 +8,15 @@ import {
 import { Spinner } from '~components';
 import { Link } from '~components/common/Link';
```

Comment:
Maybe you can also take on the circular dependencies in this one. The above is an example of a possible fix.

```import { Spinner } from '~components';``` is making the dependency circular
```import { Spinner } from '~components/common/Spinner';``` would not.

I think there are 4/5 more throughout the project. If you run ```yarn build``` the CLI output will tell you where they are.